Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using OpenQA.Selenium;
+using Mars.Drivers;
+using Mars.Helpers;
+using System;
+using dotenv.net;
+
+namespace Mars
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Load environment variables from .env file
+            DotEnv.Load();
+
+            // Initialize WebDriver
+            IWebDriver driver = WebDriverInitializer.Initialize();
```

Comment:
this should be done in hooks 

Please see
https://codebun.com/how-to-implement-hooks-in-specflow-c/